devicetypes/sinope-technologies/va4200wz-va4200zb-sinope-valve.src/va4200wz-va4200zb-sinope-valve.groovy

```diff
@@ -182,84 +182,79 @@ private boolean shouldProcessMessage(cluster) {
 	}
-    else if (descMap.cluster == "0006" && descMap.attrId == "0000") {
-        resultMap = getOnOffResult(descMap.value)
+	else if (descMap.cluster == "0006" && descMap.attrId == "0000") {
+		resultMap = getOnOffResult(descMap.value)
 	}
 	return resultMap
 }
 
 private Map getBatteryResult(rawValue) {
-    traceEvent(settings.logFilter, "Battery rawValue = ${rawValue}" + cluster, settings.trace, get_LOG_DEBUG())
+	traceEvent(settings.logFilter, "Battery rawValue = ${rawValue}" + cluster, settings.trace, get_LOG_DEBUG())
 
 	def result = [:]
-    result.name = 'battery'
-    result.descriptionText = "{{ device.displayName }} battery was {{ value }}%"
-
-    int batteryPercent = rawValue / 2
-    result.value = Math.min(100, batteryPercent)
+	result.name = 'battery'
+	result.descriptionText = "{{ device.displayName }} battery was {{ value }}%"
 
+	result.value = convertVoltToPercent(rawValue)
 	return result
 }
 
 private Map getOnOffResult(rawValue) {
-    traceEvent(settings.logFilter, "On/Off rawValue = ${rawValue}" + cluster, settings.trace, get_LOG_DEBUG())
+	traceEvent(settings.logFilter, "On/Off rawValue = ${rawValue}" + cluster, settings.trace, get_LOG_DEBUG())
 
 	Map result = [:]
-    result.name = 'valve'
-    result.descriptionText = "{{ device.displayName }} state was {{ value }}"
-    if(rawValue == "0000"){
-        result.value == "off"
-    }
-    else{
-        result.value == "on"
-    }
-    
-    List<Map> addAttribsList = []
-    Map addAttrib = [:]
-    
-    addAttrib.name = 'valve'
+	result.name = 'valve'
+	result.descriptionText = "{{ device.displayName }} state was {{ value }}"
+	if (rawValue == "0000") {
+		result.value == "off"
+	}
+	else {
+		result.value == "on"
+	}
+	
+	List<Map> addAttribsList = []
+	Map addAttrib = [:]
+	
+	addAttrib.name = 'valve'
 	addAttrib.descriptionText = "{{ device.displayName }} state was {{ value }}"
 	addAttrib.value = result.value
 	addAttribsList += addAttrib
 	result.additionalAttrs = addAttribsList
-    
+	
 	return result
 }
 
 private Map getPowerSourceResult(rawValue) {
 	traceEvent(settings.logFilter, "powerSource rawValue = ${rawValue}" + cluster, settings.trace, get_LOG_DEBUG())
 	def result = [:]
-    result.name = 'powerSource'
-    result.translatable = true
-    result.descriptionText = "{{ device.displayName }} powerSource was {{ value }}%"
-	if(rawValue == "0081" || rawValue == "0082"){
-    	result.value = "mains"
-    }
-    else if(rawValue == "0003"){
-    	result.value = "battery"
-    }
-    else if(rawValue == "0004"){
-    	result.value = "dc"
-    }
-    else{
-    	result.value = "unknown"
-    }
+	result.name = 'powerSource'
+	result.translatable = true
+	result.descriptionText = "{{ device.displayName }} powerSource was {{ value }}%"
+	if (rawValue == "0081" || rawValue == "0082") {
+		result.value = "mains"
+	}else if (rawValue == "0003") {
+		result.value = "battery"
+	}else if (rawValue == "0004") {
+		result.value = "dc"
+	}else {
```

Comment:
Since you're nicely cleaning up the code, can you add space between curly bracket and `else if`?